Sming/system/m_printf.cpp

```diff
@@ -11,39 +11,33 @@ Descr: embedded very simple version of printf with float support
 
 #define MPRINTF_BUF_SIZE 256
 
-#define OVERFLOW_GUARD 24
-
 static void defaultPrintChar(uart_t *uart, char c) {
-	return uart_tx_one_char(c);
+    return uart_tx_one_char(c);
```

Comment:
Sorry, if this is a FAQ - is there a guide regarding source code formatting for Sming (or even better, a command line tool that working with windows to do the correct formatting)? 

I tried to follow the rule 38 [here](http://geosoft.no/development/cppstyle.html#White%20Space): _Special characters like TAB and page break must be avoided.
These characters are bound to cause problem for editors, printers, terminal emulators or debuggers when used in a multi-programmer, multi-platform environment._ 

That's why I converted all tabs to space before committing.